src/store/create.ts

```diff
@@ -0,0 +1,11 @@
+import { atom } from "jotai"
+
+export const createDashboardIdAtom = atom<number | null>(null)
+export const selectedQuestionTemplateIdAtom = atom<number | null>(null)
+
+export const createQuestionKeyAtom = atom<number>(0)
+
+export const resetQuestionAtom = atom(null, (get, set) => {
+  set(createQuestionKeyAtom, get(createQuestionKeyAtom) + 1)
+  set(selectedQuestionTemplateIdAtom, null)
+})
```

Comment:
This is lifted to a global state to make it possible to reset the state from the side navigation bar.